Tests/FluentAssertions.Shared.Specs/XDocumentAssertionSpecs.cs

```diff
@@ -17,7 +17,7 @@ public class XDocumentAssertionSpecs
         #region Be / NotBe
 
         [TestMethod]
-        public void When_asserting_a_xml_document_is_equal_to_the_same_xml_document_it_should_succeed()
+        public void When_asserting_a_XDocument_is_equal_to_the_same_XDocument_it_should_succeed()
```

Comment:
I've reverted the renames of those tests and also renamed the other tests I've created to follow the same pattern.

For the question about what *equals* means in this case, I haven't looked into it. It's an existing test that I didn't modify.